desktop/cmp/treemap/TreeMap.js

```diff
@@ -0,0 +1,250 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2019 Extremely Heavy Industries Inc.
+ */
+import {Component} from 'react';
+import PT from 'prop-types';
+import {Highcharts, highchartsExporting, highchartsOfflineExporting, highchartsExportData, highchartsTree, highchartsHeatmap} from '@xh/hoist/kit/highcharts';
+
+import {XH, elemFactory, HoistComponent, LayoutSupport} from '@xh/hoist/core';
+import {div, box} from '@xh/hoist/cmp/layout';
+import {Ref} from '@xh/hoist/utils/react';
+import {resizeSensor} from '@xh/hoist/kit/blueprint';
+import {fmtNumber} from '@xh/hoist/format';
+import {assign, merge, clone, debounce} from 'lodash';
+
+import {LightTheme} from './theme/Light';
+import {DarkTheme} from './theme/Dark';
+
+import './TreeMap.scss';
+import {TreeMapModel} from './TreeMapModel';
+
+highchartsExporting(Highcharts);
+highchartsOfflineExporting(Highcharts);
+highchartsExportData(Highcharts);
+highchartsTree(Highcharts);
+highchartsHeatmap(Highcharts);
+
+/**
+ * Todo
+ */
+@HoistComponent
+@LayoutSupport
+export class TreeMap extends Component {
+
+    static propTypes = {
+        /** Primary component model instance. */
+        model: PT.oneOfType([PT.instanceOf(TreeMapModel), PT.object]).isRequired
+    };
+
+    static modelClass = TreeMapModel;
+
+    baseClassName = 'xh-treemap';
+
+    _chartElem = new Ref();
+    _chart = null;
+
+    get data() {
+        // If not bound to a grid, simply use model data
+        const {data, gridModel} = this.model;
+        if (!gridModel) return data;
+
+        // If bound to a grid, mixin selected state from GridModel
+        const selectedIds = gridModel.selModel.ids,
+            {selectionColor} = XH.darkTheme ? DarkTheme : LightTheme;
```

Comment:
I actually think the selection color on this component is both distracting and problematic.  You are actually losing the ability to visualize this critical color dimension on the one node you presumably have decided that you are interested in examining!  

Could we do a tBook style highlighting of the selected node with a border?